### PR TITLE
Added Provider AniTube (Portuguese)

### PR DESCRIPTION
**Pull request by ArjixGamer**
Added provider AniTube.

### CODE REVIEW

**Comment by ArjixGamer** [comment]
if smth is fucked up, blame git cli

**Comment by Blatzar** [comment]
Add to readme

**Comment by ArjixGamer** [comment]
> Add to readme

oh yeah

**Comment by AbdullahM0hamed** [comment]
AniTube is alright, but I personally prefer going by AniCar.

File: anime_downloader/sites/anitube.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+class AniTubeEpisode(AnimeEpisode, sitename='anitube'):
+    def _get_sources(self):
+        html = helpers.soupify(helpers.get(self.url))
+        scripts = html.find_all('script', type="text/javascript")
+
+        for script in scripts:
+            if 'var play' in str(script):
+                js = script
+
+        link = re.search(r"file.*?[\"'](.*?)['\"]", str(js)).group(1)
```
Comment (Blatzar) [inline anime_downloader/sites/anitube.py:48]: Needs better regex, see `anime dl "naruto" -p anitube -e 6 -c 1`
image

**Pull request merged**